cpp/modmesh/toggle/profile.hpp

```diff
@@ -273,7 +273,9 @@ class ScopedTimer
 /*
  * MODMESH_PROFILE defined: Enable profiling API.
  */
-#ifdef MODMESH_PROFILE
+// TODO: Migrate this into Call Profiler. See solvcon/modmesh#559.
```

Comment:
Don't do this renaming.

Reply:
Use URL `https://github.com/solvcon/modmesh/559`, not the shorthand `solvcon/modmesh#559` in code remarks.